build.sbt

```diff
@@ -18,7 +18,6 @@ val `dotty-bench-bootstrapped` = Build.`dotty-bench-bootstrapped`
 val `scala-library` = Build.`scala-library`
 val `scala-compiler` = Build.`scala-compiler`
 val `scala-reflect` = Build.`scala-reflect`
-val `dotty-repl` = Build.`dotty-repl`
```

Comment:
Is it intentional to keep the `dotty-repl` project in Build.scala?

Reply:
Looks like an oversight when rebasing.

Reply:
(Remove it in #3044)